agent/lib/kontena/load_balancers/configurer.rb

```diff
@@ -46,13 +46,15 @@ def ensure_config(container)
         end
         keep_virtual_path = env_hash['KONTENA_LB_KEEP_VIRTUAL_PATH']
         cookie = env_hash['KONTENA_LB_COOKIE']
+        basic_auth_secrets = env_hash['KONTENA_LB_BASIC_AUTH_SECRETS'].to_s
         set("#{etcd_path}/services/#{service_name}/balance", balance)
         set("#{etcd_path}/services/#{service_name}/health_check_uri", check_uri)
         set("#{etcd_path}/services/#{service_name}/custom_settings", custom_settings)
         set("#{etcd_path}/services/#{service_name}/virtual_hosts", virtual_hosts)
         set("#{etcd_path}/services/#{service_name}/virtual_path", virtual_path)
         set("#{etcd_path}/services/#{service_name}/keep_virtual_path", keep_virtual_path)
         set("#{etcd_path}/services/#{service_name}/cookie", cookie)
+        set("#{etcd_path}/services/#{service_name}/basic_auth_secrets", basic_auth_secrets) unless basic_auth_secrets.empty?
```

Comment:
Hmm... if I remove the `KONTENA_LB_BASIC_AUTH_SECRETS` from the service, then they'll just stick around in etcd?